### PR TITLE
LIME-1037 - Adding target to support link in footer

### PR DESCRIPTION
**Pull request by ElliotMurphyGDS**
### What changed
Adding target to support link in footer for accessibility

### CODE REVIEW

**Review by gds-jbentham — APPROVED** [review]

File: src/locales/en/default.yml [hunk 1]
```diff
@@ -41,7 +41,8 @@ govuk:
         - href: "https://signin.account.gov.uk/privacy-notice"
           text: "Privacy notice"
         - href: "https://home.account.gov.uk/contact-gov-uk-one-login"
-          text: "Support"
+          text: "Support (Opens in New tab)"
```
Comment (gds-jbentham) [inline src/locales/en/default.yml:44]: Should be small n for new

File: src/locales/cy/default.yml [hunk 2]
```diff
@@ -42,6 +42,7 @@ govuk:
           text: "Hysbysiad preifatrwydd"
         - href: "https://home.account.gov.uk/contact-gov-uk-one-login"
           text: "Cymorth"
+          attributes: { target: "_blank", rel:"noreferrer noopener" }
```
Comment (gds-jbentham) [inline src/locales/cy/default.yml:45]: Should welsh not also say opens in a new tab

Comment (ElliotMurphyGDS) [reply src/locales/cy/default.yml:45]: I did think this but didn't want to start messing with translations, will see if I can confirm

File: src/locales/en/default.yml [hunk 3]
```diff
@@ -41,7 +41,8 @@ govuk:
         - href: "https://signin.account.gov.uk/privacy-notice"
           text: "Privacy notice"
         - href: "https://home.account.gov.uk/contact-gov-uk-one-login"
-          text: "Support"
+          text: "Support (Opens in new tab)"
```
Comment (gds-jbentham) [inline src/locales/en/default.yml:44]: Same as fraud and DL

**Pull request merged**